### PR TITLE
SOF-6794: add more rjsf widgets

### PR DESCRIPTION
**Pull request by zoomchik**

### CODE REVIEW

File: src/other/rjsf/InputWithInfoPopover.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import InfoPopover, {
+    InfoPopoverProps,
+} from "../../mui/components/popover/info-popover/InfoPopover";
+import { PositionInfoPopover } from "./PositionInfoPopover.styled";
+
+const { BaseInputTemplate } = Templates;
+
+type InfoPopoverOptions = InfoPopoverProps & { content: string };
+
+export default function InputWithInfoPopover(props: WidgetProps) {
```
Comment (azech-hqs) [inline src/other/rjsf/InputWithInfoPopover.tsx]: I wonder if we should mark somewhere what role the components fulfill (e.g. Template, Widget, or Field). If we plan to add a couple of more RJSF components, then perhaps we can divide them into subfolders named templates, widgets, and fields. As we have only 4 right now, perhaps a simple JSDoc string would suffice. What do you think?

**Review by azech-hqs — COMMENTED** [review]
looks good 👍

File: src/other/rjsf/InputWithInfoPopover.tsx (hunk 1 above)
Comment (zoomchik) [reply src/other/rjsf/InputWithInfoPopover.tsx]: Agree, dividing into subfolders make a clean separation.

File: src/other/rjsf/InputWithInfoPopover.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    const { uiSchema } = props;
+    const infoPopover = (uiSchema ? uiSchema["ui:options"]?.infoPopover : {}) as InfoPopoverOptions;
```
Comment (k0stik) [inline src/other/rjsf/InputWithInfoPopover.tsx]: That's a very strange type casting here.
And later you use it with conditions like `infoPopover?.title` as it can be null or undefined

File: src/other/rjsf/InputWithInfoPopover.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import React from "react";
+
+import InfoPopover, {
+    InfoPopoverProps,
+} from "../../mui/components/popover/info-popover/InfoPopover";
+import { PositionInfoPopover } from "./PositionInfoPopover.styled";
+
+const { BaseInputTemplate } = Templates;
+
+type InfoPopoverOptions = InfoPopoverProps & { content: string };
```
Comment (k0stik) [inline src/other/rjsf/InputWithInfoPopover.tsx]: I found you defined this type three times in this PR. 
Why is that needed?

File: src/other/rjsf/InputWithInfoPopover.tsx (hunk 2 above)
Comment (zoomchik) [reply src/other/rjsf/InputWithInfoPopover.tsx]: Typed it more explicitly

**Review by timurbazhirov — APPROVED** [review]
